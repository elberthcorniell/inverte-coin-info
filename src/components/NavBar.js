import { Link } from 'react-router-dom';

const NavBar = () => (
    <nav>
        <Link to="/">
            <img
                alt="Inverte logo with text"
                src="https://inverte.io/images/logo-text-dark.png"
                height={40}
            />
        </Link>
        <div>
            <ul>
                <Link to='/'>
                    <li>Home</li>
                </Link>
            </ul>
        </div>
    </nav>
)

export default NavBar;
