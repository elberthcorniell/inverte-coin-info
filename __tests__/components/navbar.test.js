import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import NavBar from '../../src/components/NavBar';
import { BrowserRouter } from 'react-router-dom';

describe('test', () => {
    test('Render as spected', () => {
        render(
            <BrowserRouter>
                <NavBar />
            </BrowserRouter>
        );
        expect(document.body.innerHTML).toMatchSnapshot();
    });

});
