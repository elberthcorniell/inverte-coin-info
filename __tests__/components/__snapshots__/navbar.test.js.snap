// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Render as spected 1`] = `"<div><nav><a href=\\"/\\"><img alt=\\"Inverte logo with text\\" src=\\"https://inverte.io/images/logo-text-dark.png\\" height=\\"40\\"></a><div><ul><a href=\\"/\\"><li>Home</li></a></ul></div></nav></div>"`;
